GameServer/serverrules/AbstractServerRules.cs

```diff
@@ -2129,22 +2129,27 @@ public virtual void SendHousingMerchantWindow(GamePlayer player, DOL.GS.PacketHa
 			switch (merchantType)
 			{
 				case eMerchantWindowType.HousingInsideShop:
-					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorShopItems, eMerchantWindowType.HousingInsideShop);
+				case eMerchantWindowType.HousingInsideMenu:
+					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorShopItems, merchantType);
 					break;
 				case eMerchantWindowType.HousingOutsideShop:
-					player.Out.SendMerchantWindow(HouseTemplateMgr.OutdoorShopItems, eMerchantWindowType.HousingOutsideShop);
+				case eMerchantWindowType.HousingOutsideMenu:
+					player.Out.SendMerchantWindow(HouseTemplateMgr.OutdoorShopItems, merchantType);
 					break;
 				case eMerchantWindowType.HousingBindstoneHookpoint:
-					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorBindstoneShopItems, eMerchantWindowType.HousingBindstoneHookpoint);
+					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorBindstoneShopItems, merchantType);
 					break;
 				case eMerchantWindowType.HousingCraftingHookpoint:
-					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorCraftShopItems, eMerchantWindowType.HousingCraftingHookpoint);
+					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorCraftShopItems, merchantType);
 					break;
 				case eMerchantWindowType.HousingNPCHookpoint:
-					player.Out.SendMerchantWindow(HouseTemplateMgr.GetNpcShopItems(player), eMerchantWindowType.HousingNPCHookpoint);
+					player.Out.SendMerchantWindow(HouseTemplateMgr.GetNpcShopItems(player), merchantType);
 					break;
 				case eMerchantWindowType.HousingVaultHookpoint:
-					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorVaultShopItems, eMerchantWindowType.HousingVaultHookpoint);
+					player.Out.SendMerchantWindow(HouseTemplateMgr.IndoorVaultShopItems, merchantType);
+					break;
+				case eMerchantWindowType.HousingDeedMenu:
+					player.Out.SendMerchantWindow(/* TODO */HouseTemplateMgr.OutdoorMenuItems, eMerchantWindowType.HousingDeedMenu);

```

Comment:
I don't have any idea about what this list should contains... In this PR, I was focused to make 1.127 works.